pattern-library/sass/patterns/_alerts.scss

```diff
@@ -0,0 +1,132 @@
+// ------------------------------
+// edX Pattern Library: Utilities - Alerts
+//
+// About: Contains base styling for alerts.
+// ----------------------------
+
+// #CONFIG
+// #UTILITIES
+// #GENERAL
+// #INDIVIDUAL CASES
+
+
+// ----------------------------
+// #CONFIG
+// ----------------------------
+$alert-background-color: palette(grayscale, white) !default;
+$alert-border-radius: rem(3) !default;
```

Comment:
Should we be using a variable rather than a hard-coded value of 3? Maybe it should use one of the spacing variables:

https://github.com/edx/ux-pattern-library/blob/master/pattern-library/sass/global/_settings.scss#L305